src/test/groovy/pl/touk/service/DriverServiceSpec.groovy

```diff
@@ -0,0 +1,139 @@
+package pl.touk.service
+
+import pl.touk.model.Driver
+import pl.touk.repository.DriverRepository
+import pl.touk.service.dto.NewDriverConverter
+import pl.touk.service.dto.NewDriverDto
+import pl.touk.service.dto.UpdateDriverConverter
+import pl.touk.service.dto.UpdateDriverDto
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+
+class DriverServiceSpec extends Specification {
+
+    @Shared
+    def millisecondsInHour = 1 * 60 * 60 * 1000
+
+    def driverRepository = Mock(DriverRepository)
+    def newDriverConverter = Mock(NewDriverConverter)
+    def updateDriverConverter = Mock(UpdateDriverConverter)
+    def currentTimeService = Mock(CurrentTimeService)
+    def parkingRateService = Mock(ParkingRateService)
+
+    def service = new DriverService(driverRepository,
+            newDriverConverter, updateDriverConverter, currentTimeService, parkingRateService)
+
+    def 'should start meter for driver'() {
+        given:
+        def registrationNumber = 'WE 526'
+        def now = new Date()
+        def id = 1L
+        newDriverConverter.convert(_) >> new NewDriverDto(registrationNumber: registrationNumber, start: now)
+
+        when:
+        def newDriverDto = service.startParkingMeter(registrationNumber)
+
+        then:
+        1 * currentTimeService.now() >> now
```

Comment:
Czy chcesz to sprawdzać, dlaczego nie jest w sekcji given?

Reply:
Chce to sprawdzić w sekcji then, ponieważ tutaj określam krotność. currentTimeService.now() chce wywołać tylko raz.